src/client/redux/search/searchSelectors.js

```diff
@@ -0,0 +1,9 @@
+export const selectSearchQuery = s => {
+    return s.search.searchQuery;
+  };
+  export const selectSearchType = s => {
+    return s.search.searchType;
+  };
+  export const selectSearchResult = s => {
+    return s.search.searchResult;
+  };
```

Comment:
Declare eveything only and export all at the end.
f. ex.
`const mySelect {`
`  ...`
`}`
`const mySelect2 {`
`  ...`
`}`
`export { mySelect, mySelect2, ... }`